src/hammer-vlsi/test.py

```diff
@@ -212,6 +212,117 @@ class HammerTechnologyTest(unittest.TestCase):
     """
     Tests for the Hammer technology library (hammer_tech).
     """
+
+    def test_filters_with_extra_extraction(self) -> None:
+        """
+        Test that filters whose extraction functions return extra (non-path)
+        metadata.
+        """
+        import hammer_config
+
+        tech_dir, tech_dir_base = HammerToolTestHelpers.create_tech_dir("dummy28")
+        tech_json_filename = os.path.join(tech_dir, "dummy28.tech.json")
+
+        def add_dont_use_list(d: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
I don't get the name of this function? Is it just copy pasta?

Reply:
Yes this is accidental copy pasta. Will fix.